py/miniconf-mqtt/miniconf/__main__.py

```diff
@@ -30,10 +30,10 @@ def main():
     parser.add_argument('--no-retain', '-n', default=False,
                         action='store_true',
                         help='Do not retain the affected settings')
-    parser.add_argument('--prefix', required=True, type=str,
+    parser.add_argument('prefix', type=str,
                         help='The MQTT topic prefix of the target (or a prefix filter in the case '
                              'of discovery)')
-    parser.add_argument('settings', metavar="PATH=VALUE", nargs='*',
```

Comment:
This could still be useful when only doing `--discover` and not settings.

Reply:
True. A user could theoretically do `python -m miniconf dt/sinara/dual-iir/+ --discover -v` to detect devices.